common/src/test/scala/org/apache/celeborn/ConfigurationSuite.scala

```diff
@@ -138,8 +139,8 @@ class ConfigurationSuite extends AnyFunSuite {
   }
 
   def appendConfigurationTableHeader(output: ArrayBuffer[String]): Unit = {
-    output += "| Key | Default | Description | Since |"
-    output += "| --- | ------- | ----------- | ----- |"
+    output += "| Key | Default | Description | Since | Alternatives |"
```

Comment:
should we tip that the "Alternatives" is deprecated and will be removed in the future?

Reply:
Sounds reasonable.

Reply:
@FMX, IMO, it's recommended to introduce how to upgrade config from `Alternatives`  especially network which requires users to choose module.

Reply:
OK, I'll update this RP.

Reply:
@SteNicholas Updated. How about now?